src/pivot-table/components/helpers/get-item-size-handler.ts

```diff
@@ -42,7 +42,7 @@ export const getColumnWidthHandler =
     }
 
     // all rows except bottom one
-    if (cell?.leafCount > 0) {
+    if (!isLastRow && cell !== undefined) {
```

Comment:
"Leaf row", the row before the empty row never evaluate this to true since all cells in a "leaf row" have 0 as `leafCount`.